elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -117,6 +120,8 @@ export class PeerPool extends EventEmitter {
 		this._peerSelectForSend = peerPoolConfig.peerSelectionForSend;
 		this._peerSelectForRequest = peerPoolConfig.peerSelectionForRequest;
 		this._peerSelectForConnection = peerPoolConfig.peerSelectionForConnection;
+		this._sendPeerLimit = peerPoolConfig.sendPeerLimit === undefined ? DEFAULT_SEND_PEER_LIMIT : peerPoolConfig.sendPeerLimit;
```

Comment:
`this._sendPeerLimit = peerPoolConfig.sendPeerLimit ? DEFAULT_SEND_PEER_LIMIT : peerPoolConfig.sendPeerLimit;`

Reply:
For correctness, if the user explicitly sets it to `0`, my thinking was that it's better if `0` peers are selected; then an error will be thrown later because no peers were found.